File: Python/a006.py
coeff=input().split()
for i in range(0,len(coeff)):
  coeff[i]=int(coeff[i])

a=coeff[0]
b=coeff[1]
c=coeff[2]

d=b**2-4*a*c

if(d<0):
  print("No real root")
else:
  x1=int((-1*b+d**0.5)/2/a)
  x2=int((-1*b-d**0.5)/2/a)
  if(d==0):
    print("Two same roots x=%d" % (int(-1*b/2/a)))
  elif(d>0 and x1>x2):
    print("Two different roots x1=%d , x2=%d" % (x1,x2))
  elif(d>0 and x2>x1):
    print("Two different roots x1=%d , x2=%d" % (x2,x1))

File: C++/a006.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
 	int a,b,c,d,e;
	while(cin>>a>>b>>c)
 {
 	if(b*b-4*a*c>0)
	 {
	 	d=((-1)*b+sqrt(b*b-4*a*c))/2/a;
	 	e=((-1)*b-sqrt(b*b-4*a*c))/2/a;
	 	cout<<"Two different roots x1="<<d<<" , x2="<<e<<endl;
	 }
 	else if(b*b-4*a*c==0)
 	 {
 	 	cout<<"Two same roots x="<<(-1)*b/a/2<<endl; 
	 }
	else 
		cout<<"No real root"<<endl;
 }
    return 0;
}